src/components/ads/index.js

```diff
@@ -8,7 +8,7 @@ const Ads = () => {
           alt="By Developers for Developers"
           src="https://mdn.mozillademos.org/files/16133/Accurate_Docs_for_Precise_Development_240x480.jpg"
           style={{
-            position: 'absolute',
+            position: 'relative',
```

Comment:
here i guess `fixed` is the position we want since we want it to stick to the bottom of the viewport.
`relative` if you have a screen with a bigger height it is not shown in the bottom and doesn't look so nice

Reply:
positioning absolute or fixed seems to make outline disappear so I adjust the sidebar children to be flex-box instead. please check out the new commit